### PR TITLE
Add tabs to vaccinations page

### PR DESCRIPTION
**Pull request by misaka**
Initial go at adding tabs. Things left to do:

- [x] Fix playwright tests.
- [x] Change "Action" to "Outcome" on "Vaccinated" and "Not vaccinated" tabs
- [x] Background should be white.
- [ ] Fix tab label background
- [ ] More snagging.

image

With the flash:

image

### CODE REVIEW

File: app/controllers/vaccinations_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ class VaccinationsController < ApplicationController
   before_action :set_consent_response, only: %i[show confirm]
   before_action :set_triage, only: %i[show confirm]
 
-  layout "two_thirds"
+  layout "full_width"
```
Comment (tvararu) [inline app/controllers/vaccinations_controller.rb:10]: Isn't this equivalent to no special layout?

```suggestion
```

Comment (misaka) [reply app/controllers/vaccinations_controller.rb:10]: I thought we needed this to add the flash ... ?

Comment (benilovj) [reply app/controllers/vaccinations_controller.rb:10]: > I thought we needed this to add the flash ... ?

we shouldn't need to.

i also think we should leave this at two-thirds

Comment (misaka) [reply app/controllers/vaccinations_controller.rb]: Back to two-thirds it is!

**Review by tvararu — APPROVED** [review]

**Review by benilovj — APPROVED** [review]

**Pull request merged**